packages/plugin/src/sibling-operations.ts

```diff
@@ -1,35 +1,25 @@
 import { GraphQLFileLoader } from '@graphql-tools/graphql-file-loader';
 import { CodeFileLoader } from '@graphql-tools/code-file-loader';
 import { loadDocumentsSync } from '@graphql-tools/load';
-import { Loader, SingleFileOptions, Source } from '@graphql-tools/utils';
+import { Loader, Source } from '@graphql-tools/utils';
 import {
   FragmentDefinitionNode,
   FragmentSpreadNode,
   Kind,
   OperationDefinitionNode,
-  parse,
   SelectionSetNode,
   visit,
 } from 'graphql';
 import { ParserOptions } from './types';
 import { GraphQLConfig } from 'graphql-config';
-import { dirname } from 'path';
+import { dirname, resolve } from 'path';
 
 export type FragmentSource = { filePath: string; document: FragmentDefinitionNode };
 export type OperationSource = { filePath: string; document: OperationDefinitionNode };
 
-export const operationsLoaders: Loader<string, SingleFileOptions>[] = [
+export const operationsLoaders: Loader[] = [
   new GraphQLFileLoader(),
   new CodeFileLoader(),
-  {
-    loaderId: () => 'direct-string',
-    canLoad: async () => false,
-    load: async () => null,
-    canLoadSync: pointer => typeof pointer === 'string' && pointer.includes('type '),
-    loadSync: pointer => ({
-      document: parse(pointer),
```

Comment:
same, did we need it before?  I think it's incompatible with `graphql-tools` v7

Reply:
I think you are right 🤔